controller/secret.go

```diff
@@ -62,20 +64,31 @@ func NewSecretController(ca certmanager.CertificateAuthority, core corev1.CoreV1
 		DeleteFunc: c.deleteFunc,
 		UpdateFunc: c.updateFunc,
 	}
-	c.saStore, c.saController = cache.NewInformer(lw, &v1.ServiceAccount{}, time.Minute, rehf)
+	c.saStore, c.saController = cache.NewInformer(salw, &v1.ServiceAccount{}, time.Minute, rehf)
+
+	secretlw := &cache.ListWatch{
+		ListFunc: func(options metav1.ListOptions) (runtime.Object, error) {
+			return core.Secrets(metav1.NamespaceAll).List(options)
+		},
+		WatchFunc: func(options metav1.ListOptions) (watch.Interface, error) {
+			return core.Secrets(metav1.NamespaceAll).Watch(options)
+		},
+	}
+	c.scrtStore, c.scrtController = cache.NewInformer(secretlw, &v1.Secret{}, time.Minute, cache.ResourceEventHandlerFuncs{})
```

Comment:
So you will have a followup PR for this handlerFuncs? If so, please leave a TODO here.